packages/wdio-utils/package.json

```diff
@@ -31,6 +31,7 @@
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
   "dependencies": {
+    "@wdio/config": "^5.13.0-alpha.0",
```

Comment:
I will release all packages again, we need to change this.

Reply:
Yeap, will rebase later on